src/main/java/org/fundacionjala/coding/nestor/Movie.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.coding.nestor;
+
+public class Movie {
+    public static final int CHILDRENS = 2;
+    public static final int REGULAR = 0;
+    public static final int NEW_RELEASE = 1;
+    private String _title;
+    private int _priceCode;
+    private Price _price;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_price" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)